src/components/Button/ButtonConstants.ts

```diff
@@ -25,10 +25,9 @@ export type Size = typeof SIZES[keyof typeof SIZES] | keyof typeof OLD_BUTTON_SI
 // Support old sizes (backward compatible)
 export const getActualSize = (size?: Size) => {
   if (size && size in OLD_BUTTON_SIZES) {
-    //  Element implicitly has an 'any' type because expression of type 'Size' can't be used to index type
-    //  '{ readonly sm: "small"; readonly md: "medium"; readonly lg: "large"; }'.
-    //   Property 'xxs' does not exist on type '{ readonly sm: "small"; readonly md: "medium"; readonly lg: "large"; }'.
-    // @ts-expect-error
+    // @ts-expect-error: Element implicitly has an 'any' type because expression of type 'Size' can't be used to index type
+    //     //  '{ readonly sm: "small"; readonly md: "medium"; readonly lg: "large"; }'.
+    //     //   Property 'xxs' does not exist on type '{ readonly sm: "small"; readonly md: "medium"; readonly lg: "large"; }'.
```

Comment:
The reason we add to `@ts-expect-error`s is an opportunity to explain in human language why we put it there, copying the TS error in its entirety kinda misses that point IMO.